src/main/g8/README.md

```diff
@@ -8,13 +8,13 @@ The intention is to use `sbt new` on [this library's exterior g8 template](https
 
 # Includes... #
 
-- `Scala 2.12` for **coding**
+- `Scala 2.13` for **coding**
```

Comment:
Funny story here: I tested it out and Scala 3 works great for this project! Only blocking issue is code coverage (scoverage is still working on Scala 3 support). 

Other than that? We're golden!